### DISCUSSION TITLE
It is possible to create a BookingCustomer with Name and Surname?

### DISCUSSION THREAD

**Discussion by Loca99** [microsoftgraph/msgraph-sdk-dotnet#2525] [Q&A]
On the Microsoft site I only found this example, but the DisplayName field is inserted in the Name field:

```
BookingCustomer newCustomer = new BookingCustomer()
{
    OdataType = "#microsoft.graph.bookingCustomer",
    DisplayName = "Mario Rossi",
    EmailAddress = "fake@mail.com",
    Phones = new List<Phone>
    {
        new Phone
        {
            Number = "206-555-0100",
            Type = PhoneType.Home,
        }
    }
};

await graphClient.Solutions
    .BookingBusinesses[_bookingOptions.BookingBusinessId]
    .Customers.PostAsync(newCustomer);
```

Thank you very much!

**Comment by andrueastman** [comment]
The Api documentation only exposes the `displayName` property according to https://learn.microsoft.com/en-us/graph/api/resources/bookingcustomer?view=graph-rest-1.0#properties so it may not be possible for now. 

I would suggest providing feedback at https://aka.ms/graphfeedback to get the API team to look into supporting this one.
